### PR TITLE
Code Quality: Fixed the setting name for InfoPane

### PR DESCRIPTION
**Pull request by yaira2**
**Resolved / Related Issues**

To prevent extra work, all changes to the Files codebase must link to an approved issue marked as `Ready to build`. Please insert the issue number following the hashtag with the issue number that this Pull Request resolves.
- Closes #

**Steps used to test these changes**

Stability is a top priority for Files and all changes are required to go through testing before being merged into the repo. Please include a list of steps that you used to test this PR.

1. Opened Files ...
2. ...

### CODE REVIEW

File: src/Files.App/Views/MainPage.xaml.cs [hunk 1]
```diff
@@ -438,7 +438,7 @@ private void LoadPaneChanged()
 					(MainWindow.Instance.Bounds.Width > 450 && MainWindow.Instance.Bounds.Height > 450 || RootGrid.ActualWidth > 700 && MainWindow.Instance.Bounds.Height > 360);
 
 				ViewModel.ShouldPreviewPaneBeDisplayed = (!isHomePage || isMultiPane) && isBigEnough;
-				ViewModel.ShouldPreviewPaneBeActive = UserSettingsService.InfoPaneSettingsService.IsEnabled && ViewModel.ShouldPreviewPaneBeDisplayed;
+				ViewModel.ShouldPreviewPaneBeActive = UserSettingsService.InfoPaneSettingsService.IsInfoPaneEnabled && ViewModel.ShouldPreviewPaneBeDisplayed;
```
Comment (0x5bfa) [inline src/Files.App/Views/MainPage.xaml.cs:441]: PreviewPane to InfoPane here?

Comment (yaira2) [reply src/Files.App/Views/MainPage.xaml.cs:441]: What do you mean?

Comment (0x5bfa) [reply src/Files.App/Views/MainPage.xaml.cs:441]:
```diff
-ShouldPreviewPaneBeActive & ShouldPreviewPaneBeDisplayed
+ShouldInfoPaneBeActive & ShouldInfoPaneBeDisplayed
```

Comment (yaira2) [reply src/Files.App/Views/MainPage.xaml.cs:441]: I'd have to check the code but I think you're right.

**Pull request merged**